plugins/woocommerce/client/blocks/assets/js/blocks/category-description/edit.tsx

```
@@ -0,0 +1,97 @@
+				placeholder={ __( 'No description', 'woocommerce' ) as string }
+				value={ rawDescription }
+				onChange={ ( v: string ) =>
+					( setDescription as ( v: string ) => void )( v )
+				}
+				__experimentalVersion={ 2 }
+				{ ...blockProps }
+			/>
+		) : (
+			<p
+				{ ...blockProps }
+				dangerouslySetInnerHTML={ {
```

Biome (2.1.2)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)